fabric-events-player-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,27 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-events-player-v1",
+  "name": "Fabric Events Player (v1)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-events-player-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0",
+    "fabric-api-base": "*",
+    "fabric-mining-levels-v0": "*"
```

Comment:
Why is `fabric-mining-levels-v0` required?

Reply:
Yeah I didn't notice that one, I just copied the fabric.mod.json file